scss/tech.scss

```diff
@@ -0,0 +1,57 @@
+
+.techs-container {
+  display:flex;
```

Comment:
Remember to leave a space before the property value. Please check the whole file :smile: